### ISSUE TITLE
info box & waypoint table in separate DIVs

### ISSUE THREAD

**Issue by yakra** [TravelMapping/Web#401]
Info box & waypoint table in separate DIVs
Have scrollbar on waypoint table only
Always keep info box visible

---

By "info box", I mean the thing with
```
59 waypoints: 59 visible, 0 hidden.
0 waypoints with possible errors.
Length: 91.92 mi, 147.93 km
Average spacing: 1.58 mi, 2.55 km
Average visible spacing: 1.58 mi, 2.55 km
```

**Comment by michihdeu** [comment]
The only relevant info to me is "0 waypoints with possible errors." This should always be visible. Having as much as possible of the wp table visible is much more important than the other info.

**Comment by yakra** [comment]
> The only relevant info to me is **"0 waypoints with possible errors." This should always be visible.** Having as much as possible of the wp table visible is much more important than the other info.

Agreed. I look at the other info occasionally, but it's just candy. The "0 waypoints with possible errors." is what's important.
Thought of this due to the idea of adding in links to view unmatched/nearmatch FP data; see https://github.com/TravelMapping/Web/issues/396#issuecomment-593062712.

We could maximize space for the FP table by making it twice as wide & half as tall, and tweaking the HTML/CSS to minimize blank space above/below it.

Aw crap, don't wanna get too far into the weeds here; can't forget the `Change Layout` feature...
> Having as much as possible of the wp table visible is much more important than the other info.

@michihdeu (or anyone else), do you use this feature?

**Comment by michihdeu** [comment]
I sometimes hit the `Change Layout` button by accident and am pissed.... no, I don't need it 😉
